mypy/checker.py

```diff
@@ -553,9 +553,14 @@ def is_implicit_any(t: Type) -> bool:
                         arg_type = self.named_generic_type('builtins.tuple',
                                                            [arg_type])
                     elif typ.arg_kinds[i] == nodes.ARG_STAR2:
+                        if self.options.python_version[0] >= 3:
```

Comment:
I've got an inkling there may be other places where the same thing is needed, we just haven't found them yet. If you put the logic in unicode_type() to return str_type() if version >= 3, then you can use just unicode_type() here and we can use it in future places too.